chrome_48.txt

```diff
@@ -0,0 +1,19 @@
+[Section]
+Name = Chrome 48
+Version = 48.0.2564.109
+License = Freeware
+Description = Google Chrome is a cross-platform web browser developed by Google, based on the open source base of Chromium.
+Category = 5
+URLSite = https://www.google.com/intl/es_es/chrome/
+URLDownload = https://archive.org/download/48.0.2564.109-chrome-installer/48.0.2564.109_chrome_installer_win32.exe
+SHA1 = c25853c0fdad898c9f455375ec56821005eb3432
+SizeBytes = 44335696
+Icon = chrome.ico
+
+[Section.amd64]
+URLDownload = https://archive.org/download/48.0.2564.109-chrome-installer/chrome48_x64.zip
+SHA1 = 18a47a2285f05665e3ad19446186cfbcc30e5ba3
+SizeBytes = 80979795
+
+[Section.0a]
+Description = Google Chrome es el popular navegador web de código cerrado​ desarrollado por Google, derivado del proyecto Chromium.
```

Comment:
```suggestion
URLSite = https://www.google.com/intl/es_es/chrome/
Description = Google Chrome es el popular navegador web de código cerrado​ desarrollado por Google, derivado del proyecto Chromium.
```

Reply:
Nah, I will delete it here. Chrome always redirects to your language in your location/navigator.